README.md

```diff
@@ -383,7 +383,7 @@ terragrunt apply
 
 #### Target apply by `terragrunt`
 
-Go to layer folder `terraform/layer1-aws/` or `terraform/layer2-k8s/` and run this command:
+Go to layer folder `terraform/layer1-aws/` and run this command:
```

Comment:
It'd be better to have the same documentation for terraform too, not only for terragrunt